design/secrets.md

```diff
@@ -0,0 +1,135 @@
+# Design proposal template (replace with your proposal's title)
+
+Currently, Velero only supports a single AIM access credential secret per location provider/plugin. Velero creates and stores the plugin credential secret under the hard-coded key `secret.cloud-credentials.data.cloud`.
```

Comment:
`AIM` -> `IAM`, though I think we can actually say "single credential secret per platform."